API/user/get.js

```diff
@@ -3,6 +3,7 @@ const User = Models.user;
 
 const getUser = async (request, response) => {
   response.status(200).json(await User.findAll());
+  const wtf = await User.findAll();
```

Comment:
Is this a used variable?

Reply:
Not used at all. It was made for testing purposes, I forgot to delete it.